libs/UpdatePackage.php

```diff
@@ -157,6 +158,31 @@ public function checkPhpVersion()
         return $newMinPhpVersion;
     }
 
+    /**
+     * Check for restricted modules
+     *
+     * @return string|true TRUE - if no restriction, String - error message
+     */
+    public function checkRestrictedModules()
+    {
+        // Conditions when modules should be restricted:
+        $restrictedModules = [
+            'enterprise' => ['HumHubVersion', '>=', '1.10']
```

Comment:
Wouldn't it be good to include the error message in the array? In case further restrictions are added later?

Reply:
@luke- Done in commit https://github.com/humhub/updater/pull/32/commits/3e7d5e0b6b9c0b0c97efbefe28edc5be6c02c080.